hooks/use-css-vars/src/generate-css-vars.ts

```diff
@@ -18,7 +13,8 @@ const isEmptyObject = (obj: Value) => {
 
 export const generateCSSVarTuples: GenerateCSSVarTuples = (theme, prefix) => {
   let acc: CSSVarsArray = [];
-  const traverseObject: TraverseObject = (v, pre = prefix ? `--${prefix}-` : '-') => {
```

Comment:
Was `TraverseObject` since it was only used wtihin the scope of this function?

Reply:
+ TS was complaining about some of the syntax, yes.

Reply:
I decided to clean up that types file too. I removed all the types that are not actually used across functions. Much cleaner this way, I think.